SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -627,3 +196,23 @@ void USpatialLatencyTracer::Debug_SendTestTrace()
 	});
 #endif // TRACE_LIB_ACTIVE
 }
+
+void USpatialLatencyTracer::SetWorkerId(const FString& NewWorkerId)
+{
+#if TRACE_LIB_ACTIVE
+	if (Tracer.IsValid())
```

Comment:
Nit: It's safe to assume this is valid, the lifetime is tied to USpatialLatencyTracer